gcp_variant_transforms/pipeline_common.py

```diff
@@ -54,6 +58,11 @@
     '--range_partitioning=start_position,0,{TOTAL_BASE_PAIRS},{PARTITION_SIZE} '
     '--clustering_fields=start_position,end_position '
     '{FULL_TABLE_ID} {SCHEMA_FILE_PATH}')
+_BQ_LOAD_AVRO_COMMAND = (
+    'bq load --source_format=AVRO {FULL_TABLE_ID} {AVRO_FILE_BASE_NAME}-*')
```

Comment:
We should discuss the relative merits of this versus making API requests and skipping the intermediate files altogether.